Auto merge dependabot dev dependency PRs

notlee opened the pull request.

Commit:
Auto merge dependabot dev dependency PRs

https://github.com/Financial-Times/origami/issues/55

JakeChampion commented:
hey @Financial-Times/ig-developers , we're auto-merging dev dependencies which are bumped by dependabot in working hours if tests pass: https://github.com/Financial-Times/origami/issues/55

I opened a PR on all Origami things, including those maintained by other teams. Feel free to close this PR if you wouldn't like this for some reason!

Commit:
Add Github Action to auto approve dependabot

So it may auto-merge dev dependencies: https://github.com/Financial-Times/origami/issues/55

notlee committed:
Update .github/workflows/auto-approve.yml

notlee committed:
Merge pull request #15 from Financial-Times/dependabot-autoapprove

Add Github Action to auto approve dependabot

notlee commented:
origami maintained components have moved to Dependabot native now, so closing this PR
